### PR TITLE
Compile CSS with autoprefixer

### PR DESCRIPTION
**Pull request by notlee**
This is required by Origami components, even for modern browsers,
due to the use of the `appearance` property.

Screenshot 2020-05-20 at 17 42 28

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   "main": "./lib/service.js",
   "scripts": {
     "start": "heroku-node-settings index.js",
-    "build:css": "mkdir -p public && sass src/main.scss public/main.css --style=compressed --load-path=bower_components --load-path=.",
+    "build:css": "mkdir -p public && sass src/main.scss  --style=compressed --load-path=bower_components --load-path=. | postcss --use autoprefixer --output public/main.css --map",
```
Comment (JakeChampion) [inline package.json:25]: do we have a browserslist config for the autoprefixer to use or are we using the default values?

Comment (notlee) [reply package.json:25]: I've added IE11 to a browserlist, which is the main one

Comment (notlee) [reply package.json:25]: although we never really test the registry in IE11, and I don't care to go to much extra effort for it for the registry

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v217.0.5 :tada:

**Pull request merged**